sources/packages/web/src/views/institution/locations/request-a-change/RequestAChangeFormSubmit.vue

```diff
@@ -67,6 +67,7 @@
         primaryLabel="Submit requested change"
         @primaryClick="submit"
         @secondaryClick="cancel"
+        :showButtons="!isReadOnlyUser(locationId)"
```

Comment:
Similar to this one: https://github.com/bcgov/SIMS/pull/4325#discussion_r1947329388